thunder/core/prims.py

```diff
@@ -2814,6 +2815,14 @@ def _empty_meta(
 empty = make_prim(PrimIDs.EMPTY, "empty", meta=_empty_meta)
 
 
+# TODO(crcrpar): Cover `memory_format` kwarg
+def _clone_meta(a: TensorProxy, **kwargs) -> TensorProxy:
+    return TensorProxy(like=a, requires_grad=a.requires_grad)
```

Comment:
requires_grad attribute is only for input TensorProxy and it's unused for intermediates. There's no harm in setting it, but this could be just `TensorProxy(like=a)`.
Was anything breaking without the requires_grad set?